LiskHQ/lisk-hub
# Add remaining request steps - Closes #782

### What was the problem?
We didn't have the remaining request lsk steps implemented yet

### How did I fix it?
Added the steps

### How to test it?
Go through the steps, check on mobile as well, switch between send & request a few times

### Review checklist
- The PR solves #782
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)
